### ISSUE TITLE
Build deb packages for arm

### ISSUE THREAD

**Issue by ilovethensa** [netbirdio/netbird#1239]
**Is your feature request related to a problem? Please describe.**
I cant install netbird on my rasberry pi since there arent deb packages for arm

**Describe the solution you'd like**
build deb packages for arm

**Describe alternatives you've considered**
Not using netbird

**Additional context**
no context needed

**Comment by mlsmaycon** [comment]
Hello @ilovethensa, NetBird has CLI packages for ARM systems. You can install it with the following steps:

With the convenience script:
```bash
export SKIP_UI_APP=true
curl -fsSL https://pkgs.netbird.io/install.sh | sh
```

Manual steps:

1. Add repository:
```bash
sudo apt-get update
sudo apt-get install ca-certificates curl gnupg -y
curl -sSL https://pkgs.netbird.io/debian/public.key | sudo gpg --dearmor --output /usr/share/keyrings/netbird-archive-keyring.gpg
echo 'deb [signed-by=/usr/share/keyrings/netbird-archive-keyring.gpg] https://pkgs.netbird.io/debian stable main' | sudo tee /etc/apt/sources.list.d/netbird.list
```
2. Update APT's cache
```bash
 sudo apt-get update
```
3. Install the package
```bash
 sudo apt-get install netbird
 ```

**Comment by alexhaydock** [comment]
It's good that the installation of the CLI package can complete even on an aarch64 system, but with the increasing stabilisation of Asahi Linux at daily-driver quality as well as more ARM devices hitting the market, I think it's worth investigating whether aarch64 builds of the `netbird-ui` can be enabled.

**Comment by nazarewk** [comment]
I thinks this might be already handled, but keeping the issue opened until it's explicitly verified.

**Comment by TheScreechingBagel** [comment]
i don't believe they are, e.g. [the latest release](https://github.com/netbirdio/netbird/releases/tag/v0.61.2) has only `netbird_0.61.2_linux_arm64` releases, not `netbird-ui_0.61.2_linux_arm64`

and the https://pkgs.netbird.io/yum/ repo didn't seem to have aarch64 build either when i tried

**Comment by PizzaLovingNerd** [comment]
I am working on this, pull request shortly.
https://github.com/PizzaLovingNerd/netbird/tree/ui-arm-build
